gnd/src/main/java/com/google/android/gnd/ui/terms/TermsOfServiceViewModel.java

```diff
@@ -16,9 +16,31 @@
 
 package com.google.android.gnd.ui.terms;
 
+import androidx.lifecycle.MutableLiveData;
+import com.google.android.gnd.persistence.local.LocalValueStore;
+import com.google.android.gnd.rx.annotations.Hot;
 import com.google.android.gnd.ui.common.AbstractViewModel;
+import com.google.android.gnd.ui.common.Navigator;
+import javax.inject.Inject;
 
 // TODO: Needs to handle view state and behaviors of the Terms Fragment
 public class TermsOfServiceViewModel extends AbstractViewModel {
 
+  private final Navigator navigator;
+  private final LocalValueStore localValueStore;
+
+  @Hot(replays = true)
+  public final MutableLiveData<Boolean> termsCheckBox = new MutableLiveData<>();
```

Comment:
This represents the check boxes' selection state, not the checkbox itself, so should be named accordingly (eg `acceptTermsCheckboxState`)

Reply:
updated in https://github.com/google/ground-android/pull/845